source/customizing_look_and_feel/02_configuring_navigation/navigation.rst

```diff
@@ -1,7 +1,66 @@
-Configuring the navigation menu
-===============================
+Configuring the navigation
+==========================
 
-Once your page content has been fully defined, you might want to *plug* your page in your domain's menu. This can easily
-be done by editing the menu components in the *Domain > Portal > General settings* section.
+The navigation in an Opendatasoft portal starts with the root page, that welcomes users, and is then mainly guided by the navigation header located at the top of any interface in the front office. Both are configurable via the **Look and feel > Navigation** subsection of the back office.
```

Comment:
"The navigation in an Opendatasoft portal starts with the root page" is unclear.